src/index.js

```diff
@@ -84,12 +84,14 @@ const composeAssetClip = ({clipName, ref, start, duration, audioRole = 'dialogue
 const jsonToFCPX = (sequenceEDLJson)=>{
     let counterIdAssetClips = 1;
     // sequenceEDLJson.events 
-    const assetClips = sequenceEDLJson.events.map((event, index)=>{
-        const results = composeAssetClip({
+    const assetClips = sequenceEDLJson.events.map((event, index) => {
+        const durationFrames = (event.endTime - event.startTime) * 2500;
+        const roundedDuration = Math.ceil(durationFrames / 100) * 100; // removes clip boundary warning on FCPX import
```

Comment:
Any reason to use Math.ceil here rather than Math.round? I trust your judgement!

Reply:
For some reason, that I am not 100% clear on, duration needs to be divisible by 100 or it gives a warning when you import it into FCPX. This is a very unsatisfactory answer I know, but maybe I'll figure it out one day!

Reply:
I've now re-read this and realise what you mean, will use Math.round instead!